src/storage/disk_manager.c

```diff
@@ -4939,9 +4957,15 @@ disk_manager_init (THREAD_ENTRY * thread_p, bool load_from_disk)
     {
       disk_Temp_max_sects = SECTID_MAX;	/* infinite */
     }
+  else if (disk_Temp_max_sects == 0)
+    {
+      disk_Temp_max_sects = 0;
+    }
   else
     {
-      disk_Temp_max_sects = disk_Temp_max_sects / DISK_SECTOR_NPAGES;
+      disk_Temp_max_sects < (DISK_MIN_VOLUME_SECTS * DISK_SECTOR_NPAGES) ? (DISK_MIN_VOLUME_SECTS * DISK_SECTOR_NPAGES) : disk_Temp_max_sects;
```

Comment:
Did you mean to do this?
```suggestion
      disk_Temp_max_sects = disk_Temp_max_sects < (DISK_MIN_VOLUME_SECTS * DISK_SECTOR_NPAGES) ? (DISK_MIN_VOLUME_SECTS * DISK_SECTOR_NPAGES) : disk_Temp_max_sects;
```

Reply:
or you can do like this
```suggestion
      disk_Temp_max_sects = MAX (DISK_MIN_VOLUME_SECTS * DISK_SECTOR_NPAGES, disk_Temp_max_sects);
```